src/client/fuse_ll.cc

```diff
@@ -136,13 +136,21 @@ static int getgroups_cb(void *handle, gid_t **sgids)
   return getgroups(req, sgids);
 }
 
-#define GET_GROUPS(perms, req)	{				\
-  if (g_conf->get_val<bool>("fuse_set_user_groups")) {	\
-    gid_t *gids = NULL;						\
-    int count = getgroups(req, &gids);				\
-    perms.init_gids(gids, count);				\
-    perms.take_gids();						\
-  } }
+static int get_fuse_groups(UserPerm& perms, fuse_req_t req)
```

Comment:
returns `int` ?

Reply:
Sorry, I got it. :-)

Reply:
No, you're right. That should be void return. Will fix.

Reply:
@jtlayton It was not there in the previous jenkins build. I'm not sure why. 

But I got it in my local build:
```
$ make client/fuse_ll.cc.o
/home/jcollin/workspace/ceph/src/client/fuse_ll.cc: In function ‘int get_fuse_groups(UserPerm&, fuse_req_t)’:
/home/jcollin/workspace/ceph/src/client/fuse_ll.cc:153:1: warning: no return statement in function returning non-void [-Wreturn-type]
 }
 ^
```